src/Repository/AdherentRepository.php

```diff
@@ -512,4 +513,16 @@ public function findAdherentsUuidByEmailAddress(string $emailAddress): array
             return $uuid->toString();
         }, array_column($query->getArrayResult(), 'uuid'));
     }
+
+    public function countByGenderAndTotal(): array
+    {
+        $qb = $this->createQueryBuilder('a')
+            ->select('COUNT(a.id) AS total, SUM(case WHEN a.gender = :female then 1 else 0 end) AS female, SUM(case WHEN a.gender = :male then 1 else 0 end) AS male')
+            ->andWhere('a.adherent = 1')
+            ->setParameter('female', Genders::FEMALE)
+            ->setParameter('male', Genders::MALE)
+        ;
+
+        return $qb->getQuery()->getResult()[0];
```

Comment:
We can call `getSingleResult()` instead.

Reply:
No more need after refactoring https://github.com/EnMarche/en-marche.fr/pull/2358/files#diff-e7a7adb167b1d2e64b02883d2be7eff1R518.